ethereum_history_api/oracles/src/noir/oracles/common/encode.test.ts

```diff
@@ -118,16 +118,21 @@ describe('encodeAddress', () => {
 
   it('mix case', () => {
     // prettier-ignore
-    expect(encodeAddress('0xb47e3cd837dDF8e4c57f05d70ab865de6e193bbb')).toStrictEqual([
-      '0xb4', '0x7e', '0x3c', '0xd8', '0x37', '0xdD', '0xF8', '0xe4', '0xc5', '0x7f', '0x05', '0xd7', '0x0a', '0xb8',
-      '0x65', '0xde', '0x6e', '0x19', '0x3b', '0xbb'
+    expect(encodeAddress('0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB')).toStrictEqual([
```

Comment:
`0xb47e3cd837dDF8e4c57f05d70ab865de6e193bbb` has an incorrect checksum and previous version of `viem` `isAddress` didn't checked this checksum.